### DISCUSSION TITLE
Restore acid rain?

### DISCUSSION THREAD

**Discussion by Firestorm01X2** [cataclysmbn/Cataclysm-BN#2063] [General]
Acid rains seems still supported and can be enabled in regional_map_settings,json.
Over history acid rains were heavily nerfed already and should not pose substantional problems. And the same time they may add variety to weather condition with a little work of enabling it via json config.

Should we go for it?

**Comment by chaosvolt** [comment]
If they don't have the old weird problem of murdering every monster outdoors then it should be fine.

**Comment by Firestorm01X2** [comment]
> If they don't have the old weird problem of murdering every monster outdoors then it should be fine.

https://github.com/CleverRaven/Cataclysm-DDA/pull/13727
https://github.com/CleverRaven/Cataclysm-DDA/pull/454

I guess it should not

**Comment by chaosvolt** [comment]
That fixes it melting items, and the player doesn't take direct damage it seems. What happens to monsters caught out in it?

**Comment by Firestorm01X2** [comment]
![изображение](https://user-images.githubusercontent.com/17512620/153266361-444d1034-ec7d-452b-8c62-0a2dbefcb851.png)
Crow passed the test. 3 min - no health damage

**Comment by chaosvolt** [comment]
Seems like that does fix most of the issues that warranted acid rain getting the boot, then.

If we ever get z-level trees (or just fake it by assuming any tile next to an adult tree has at least a basic level of canopy cover to reduce exosure) plus monsters smart enough to take shelter, then having it debuff or harm monsters again would be interesting.

**Comment by Zlorthishen** [comment]
Acid rain would be great to have back, but it should come with some kind of warning "The clouds are getting an ominous green look", ect

**Comment by JonathanLochridge** [comment]
Sounds like a cool idea. Although, having a toggle for it would be even better in my opinion. I don't have an issue with acid rain being the default. Assuming the occurance rate isn't too high. Not sure how you add a toggle to the options menu though?
